core/org.wso2.carbon.tomcat.ext/src/main/java/org/wso2/carbon/tomcat/ext/valves/RequestCorrelationIdValve.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.log4j.MDC;
+import org.wso2.carbon.tomcat.ext.config.TraceIdConfiguration;
```

Comment:
This class should have been deleted now

Reply:
Fixed with 26e75d7c4871fd6caa4f5149c93f18e78771bbc3